File: codigos/1 - estrutura sequencial/1008 - Salario/Main.c
// Online C compiler to run C program online
#include <stdio.h>

int main() {
    int number;
    int horas;
    double salario_hora;
    
    scanf("%d\n%d\n%lf", &number, &horas, &salario_hora);
    
    printf("NUMBER = %d\n", number);
    printf("SALARY = U$ %.2lf\n", horas*salario_hora);
    
    return 0;
}

File: codigos/1 - estrutura sequencial/1008 - Salario/Main.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var prompt = function(texto) { return lines.shift();};
var A = parseInt(prompt("Escreva o numero do funcionario"));
var B = parseInt(prompt("Escreva o numero de horas trabalhadas pelo funcionario"));
var C = parseFloat(prompt("Escreva o o valor que o funcionario recebe por hora trabalhada"));
var D = (B*C).toFixed(2);
W = `NUMBER = ${(A)}
SALARY = U$ ${(D)}`;
console.log(W);